components/style/traversal.rs

```diff
@@ -149,6 +149,12 @@ pub trait DomTraversalContext<N: TNode>  {
     /// Process `node` on the way up, after its children have been processed.
     fn process_postorder(&self, node: N);
```

Comment:
Add a comment indicating that this is only called if has_postorder_traversal returns true.